### ISSUE TITLE
The language setting is garbled when set to Chinese.

### ISSUE THREAD

**Issue by fantastic-d5** [ATLauncher/ATLauncher#953]
### What happened?

The language setting only has a garbled option available.
<img width="403" alt="iShot_2024-09-02_10 41 14" src="https://github.com/user-attachments/assets/cab9d3d7-7eb5-4e22-b78d-08333cbd3d86">
Unable to access the folder under the user Desktop directory.
<img width="513" alt="iShot_2024-09-02_10 50 52" src="https://github.com/user-attachments/assets/d884c290-d760-4432-8d6d-3d4d0b33846d">

### Operating System

macOS

### Relevant log output

```shell
macos 11.7.10


[02/09/2024 10:41:59 上午] Organising filesystem
[02/09/2024 10:41:59 上午] ATLauncher Version: 3.4.37.2 [f2a276fe8633bb138f6434bd4650544800bd7dc2]
[02/09/2024 10:41:59 上午] App Arguments: []
[02/09/2024 10:41:59 上午] JVM Arguments: ["-Duser.dir=**USERSDIR**","-Dapple.laf.useScreenMenuBar=true","-Djna.nosys=true","-Djava.net.preferIPv4Stack=true","-Dawt.useSystemAAFontSettings=on","-Dswing.aatext=true"]
[02/09/2024 10:41:59 上午] Java Version: Java 8 (1.8.0_291)
[02/09/2024 10:41:59 上午] Java Path: /Library/Java/JavaVirtualMachines/jdk1.8.0_291.jdk/Contents/Home/jre
[02/09/2024 10:41:59 上午] 64 Bit Java: true
[02/09/2024 10:41:59 上午] RAM Available: 8192MB
[02/09/2024 10:41:59 上午] Launcher Directory: **USERSDIR**
[02/09/2024 10:41:59 上午] Using Mac App? Yes
[02/09/2024 10:42:00 上午] GPU: Intel UHD Graphics 617 (Intel) Revision ID: 0x0002 1536MB VRAM
[02/09/2024 10:42:00 上午] CPU: Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz 2 cores/4 threads
[02/09/2024 10:42:00 上午] Operating System: macOS (11.7.10 (Big Sur) build 20G1427)
[02/09/2024 10:42:00 上午] Bitness: 64
[02/09/2024 10:42:00 上午] Uptime: 4555960
[02/09/2024 10:42:00 上午] Manufacturer: Apple
[02/09/2024 10:42:00 上午] Showing splash screen and loading everything
[02/09/2024 10:42:00 上午] Checking for updated files!
[02/09/2024 10:42:03 上午] Launcher finished loading everything
[02/09/2024 10:42:03 上午] Launcher opening
[02/09/2024 10:42:03 上午] Made By Bob*
[02/09/2024 10:42:03 上午] *(Not Actually)
[02/09/2024 10:42:03 上午] Setting up Bottom Bar
[02/09/2024 10:42:03 上午] Finished Setting up Bottom Bar
[02/09/2024 10:42:03 上午] Setting up Tabs
[02/09/2024 10:42:03 上午] Finished Setting up Tabs
[02/09/2024 10:42:03 上午] Showing Launcher
```
```


### Additional Information

_No response_

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fantastic-d5** [comment]
<img width="806" alt="iShot_2024-09-02_10 53 49" src="https://github.com/user-attachments/assets/23a81d22-948d-4ba0-ba66-365de6630f25">

**Comment by RyanTheAllmighty** [comment]
Yes this is a known issue and will be solved once we have implemented multi language support #640

**Comment by fantastic-d5** [comment]
Okay, thank you.
